src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -2066,7 +2066,7 @@ private void SendLeftArrows(int length)
                 up.Data.Keyboard.ExtraInfo = IntPtr.Zero;
 
                 inputs[2 * i] = down;
-                inputs[2 * i + 1] = up;
+                inputs[(2 * i) + 1] = up;
```

Comment:
I'd wonder if anybody wanted parentheses in such expressions.

Reply:
Multiplicative > Additive